azure/sdk-identity-resources-storage/README.md

```diff
@@ -0,0 +1,71 @@
+﻿## Prerequisites
+
+* An [Azure subscription]().
+
+## Preparing to run the sample
+
+Use the [Azure Cloud Shell](https://shell.azure.com) to create/get client secret credentials.
+
+* Create a service principal and configure its access to Azure resources:
```

Comment:
Convert to a numbered list, since these steps must be completed in the order listed.